openreview/venue_request/process/deployProcess.py

```diff
@@ -615,4 +615,8 @@ def process(client, note, invitation):
                 }
             },
             signatures=['~Super_User1']
-        ))
\ No newline at end of file
+        ))
+
+        # Add API 2 venues to active venues group
+        active_venues = client.get_group('active_venues')
```

Comment:
I think this change should be done inside of the Venue class so the venue is added to the group even if we don't use the venue request form.

please remove the code where we are adding in the post submission stage.

Reply:
> so the venue is added to the group even if we don't use the venue request form.

what does this mean? don't all venues have to submit a venue request form?

Reply:
you can create a venue using the Venue class directly, no need to create a request form. The Venue class should be cohesive and have all the behavior inside, if you check the code we are already adding the venue to active_venues inside of the Venue class but only when post_submission_stage is run manually. I think you should move that line and run it in the setup function.